packages/web/pages/stake/index.tsx

```diff
@@ -268,6 +274,41 @@ export const Staking: React.FC = observer(() => {
 
   const showStakeLearnMore = !isWalletConnected || isNewUser;
 
+  const { unbondingBalances } = unbondingDelegationsQuery;
+  const unbondingInProcess = unbondingBalances.length > 0;
+
+  function groupByCompletionTime(
```

Comment:
As mentioned in my slack message, I think this is the only way to get a single stake object